homeassistant/components/somfy/__init__.py

```diff
@@ -58,6 +60,10 @@
     if DOMAIN not in config:
         return True
 
+    hass.data[DOMAIN][CONF_OPTIMISTIC] = False
+    if CONF_OPTIMISTIC in config[DOMAIN]:
```

Comment:
To double check, but as you set a default value for the [conf](https://github.com/home-assistant/home-assistant/pull/31407/files#diff-5f6fcaa13fcf30fc2741aedd6cc2666bR46), this condition is useless.

Reply:
removed redundant check.